test/jdk/sun/security/provider/PolicyFile/Comparator.java

```diff
@@ -31,6 +31,9 @@
  * Note:  if you want to see the java.security.debug output,
  *        you can not simply set the system property.
  *        you must run this test by hand and pass -Djava.security.debug=...
+ *
+ * Note: SubjectCodeSource removed, but PolicyFile might have the same problem. Keep the test as-is
+ *
```

Comment:
I may have the note in a bug description instead.  Is it safe to remove this test and related files?

Reply:
@XueleiFan I have removed the Test and related policy files
(Comparator.Combined.Policy/ Comparator.Comparator.Policy/Comparator.Principal.Policy), is this what you were expecting? I can confirm I am still getting a successful test run.

Reply:
I meant to remove the Comparator.java.   As SubjectCodeSource has been removed, the note you added may be confusing for future readers.  If you want to have a record, you could use bug comment in JDK-8294526.

Reply:
Thanks, I have removed the Comparator.java and related files and will add a comment on the bug